autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -137,10 +138,15 @@ def run(self):
             allow_methods=["*"],  # Allows all methods
             allow_headers=["*"],  # Allows all headers
         )
+        log_config = dict(uvicorn.config.LOGGING_CONFIG)
```

Comment:
Can't we move this replicated logic to configure_logging?
This kind of hinting that we also need this on all services / process/

Reply:
Replicated logic moved